Mathlib/Topology/MetricSpace/HausdorffDistance.lean

```diff
@@ -591,6 +591,14 @@ theorem _root_.IsClosed.not_mem_iff_infDist_pos (h : IsClosed s) (hs : s.Nonempt
   simp [h.mem_iff_infDist_zero hs, infDist_nonneg.gt_iff_ne]
 #align is_closed.not_mem_iff_inf_dist_pos IsClosed.not_mem_iff_infDist_pos
 
+-- porting note: new lemma
+theorem continuousAt_inv_infDist_pt (h : x ∉ closure s) :
```

Comment:
Is thisre precedence for this `pt` name?

Reply:
Yes, we have `continuous_infDist_pt`. Probably, we also have (at least, in Lean 3) a lemma saying that it's continuous both in pt and set.